core-management/core-management-subsystem/src/main/java/org/wildfly/extension/core/management/CoreManagementRootResourceDefinition.java

```diff
@@ -26,7 +42,7 @@ class CoreManagementRootResourceDefinition extends PersistentResourceDefinition
     CoreManagementRootResourceDefinition() {
         super(CoreManagementExtension.SUBSYSTEM_PATH,
                 CoreManagementExtension.getResourceDescriptionResolver(),
-                ModelOnlyAddStepHandler.INSTANCE,
+                new CoreManagementAddHandler(),
```

Comment:
Since the logic within this add operation handler is specific to the unstable API child resource, why register it here rather than on the child resource itself?